packages/slice-machine/src/apiClient.ts

```diff
@@ -35,6 +38,22 @@ export const saveCustomType = (
   return axios.post("/api/custom-types/save", requestBody, defaultAxiosConfig);
 };
 
+export const renameCustomType = (
+  customTypeId: string,
+  newCustomTypeName: string
+): Promise<AxiosResponse> => {
+  const requestBody: RenameCustomTypeBody = {
+    customTypeId,
+    newCustomTypeName,
+  };
+
+  return axios.patch(
```

Comment:
I think this is the first time I've seen some-one use `patch`

Reply:
we are only changing part of the element (only the name) so I guess it's a `patch` rather than a `put` yeah?